src/frontend/store/actions/show-modal.ts

```diff
@@ -0,0 +1,24 @@
+import { ConfirmModalData } from '../../interfaces'
+import { ModalMessageInState } from '../store'
+
+export const SHOW_MODAL = 'SHOW_MODAL'
+
+export type ModalResponse = {
```

Comment:
`ConfirmModalResponse`

change file name to `show-confirm-modal.ts`

Reply:
changed